README.md

```diff
@@ -1 +1,43 @@
-# SAP BTP Best Practices
\ No newline at end of file
+# SAP BTP Best Practices
+
+## PREREQUISITES
+
+1. Setup [ABAPGit](https://docs.abapgit.org/user-guide/getting-started/install.html#prerequisites).
```

Comment:
Put all of these in the README file under the abap folder. This README file here is the overall standards in BTP -- abap and non-abap, development and non-development standards.